File: aula1003.js
/////////////////////////////////////////
//       AULA 05 - 10.03.2025         //
///////////////////////////////////////

// ATIVIDADE 01 //

let preco = 150;
let desconto = 20;
let preco_final = preco * (desconto / 100);
console.log('O preço final é', preco_final);

// ATIVIDADE 02 //

let peso = 70;
let altura = 1.75;
let imc = peso / (altura ** 2);
console.log('O Índice de Massa Corporal vale', imc);

// ATIVIDADE 03 //

let idade = 22;
let tem_exp = true;
let concurso_eleg = idade >= 18 && tem_exp;
console.log('É elegível para o concurso?');
if (concurso_eleg) {
    console.log('Sim!')
} else {
    console.log('Não!')
}

// ATIVIDADE 04 //

let nivel_acesso = parseInt(prompt('Digite seu nível de acesso'));
if (nivel_acesso === 3) {
    console.log('Acesso total!')
} else if (nivel_acesso === 2) {
    console.log('Acesso parcial!')
} else if (nivel_acesso === 1) {
    console.log ('Acesso restrito!')
} else {
    console.log('Erro. Digite um número válido!')
}

// ATIVIDADE 05 //

let c = parseFloat(prompt('Qual a temperatura no momento?'))
let f = (((c * 9) / 5) + 32)
console.log('A temperatura marca ' + c + 'ºC ou ' + f + 'ºF!')

File: UC1/aula1003.py
########################################
#      AULA 05 - 10.03.2025          ##
######################################

# ATIVIDADE 01

preco = 150
desconto = 20
print('O preço final é ', preco - desconto)
 
# ATIVIDADE 02

peso = 70
altura = 1.75
imc = (peso / (altura ** 2))
print('O Índice de Massa Corporal é igual a {}!'.format(imc))

# ATIVIDADE 03

idade = 22
tem_exp = True
concurso_eleg = idade >= 18 and tem_exp
print('Essa pessoa é elegível para o concurso?')
if concurso_eleg:
    print('Sim!')
else:
    print('Não!')

# ATIVIDADE 04

nivel_acesso = int(input('Digite seu nível de acesso:'))
if nivel_acesso == 3:
    print('Acesso total!')
elif nivel_acesso == 2:
    print('Acesso parcial!')
elif nivel_acesso == 2:
    print('Acesso restrito!')
else:
    print('Erro. Digite um número válido!')

# ATIVIDADE 05

c = float(input('Qual a temperatura, em ºC, no momento?'))
f = (((c * 9) / 5) + 32)
print('A temperatura marca {}ºC ou {}ºF!'.format(c, f))

# ATIVIDADE 06

num = int(input('Digite um número:'))
par = (num % 2)
if par == 0:
    print(num,': Este é um número par!')
else:
    print(num,': Este é um número ímpar!')

# ATIVIDADE 07

peso = int(input('Quanto pesa o pacote em kg?'))
if peso <= 5:
    print('Para {}kg, frete: R$10,00'.format(peso))
elif 5 < peso <= 10:
    print('Para {}kg, frete: R$20,00'.format(peso))
elif peso > 10:
    print('Para {}kg, frete: R$30,00'.format(peso))
